templates/quickstart-confluence-master-with-vpc.template.yaml

```diff
@@ -517,8 +517,8 @@ Parameters:
     Type: String
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
-    Description: The EC2 Key Pair to allow SSH access to the instances. 
-    Type: List<AWS::EC2::Keypair>
+    Description: The EC2 Key Pair to allow SSH access to the instances.
+    Type: List<AWS::EC2::Keypair::KeyName>
```

Comment:
`AWS::ec2::KeyPair::KeyName` returns an object of all available KeyPairs so no need to wrap in a List. Should be defined like so:

`Type: AWS::ec2::KeyPair::KeyName`

Reply:
Done!